mdsuite/experiment/run.py

```diff
@@ -93,14 +94,14 @@ def transformation_wrapper(self, func: Union[Type[Transformations], Any]):
         func: a transformation to be attached to the experiment/s
         """
 
-        @functools.wraps(func)
+        @functools.wraps(func.run_transformation)
```

Comment:
Have we checked that the wraps works properly with autocompletion on the transformations?

Reply:
no